apps/etterlatte-brev-api/src/test/kotlin/no/nav/etterlatte/brev/model/EtterbetalingBrevTest.kt

```diff
@@ -0,0 +1,58 @@
+package no.nav.etterlatte.brev.model
+
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.shouldNotBe
+import no.nav.etterlatte.brev.behandling.Beregningsperiode
+import no.nav.etterlatte.libs.common.beregning.BeregningsMetode
+import no.nav.pensjon.brevbaker.api.model.Kroner
+import org.junit.jupiter.api.Test
+import java.time.LocalDate
+
+internal class EtterbetalingBrevTest {
+    @Test
+    fun `skal kun returnere perioder som er innenfor etterbetalingsdatoer`() {
+        val etterbetalingDto =
+            EtterbetalingDTO(
+                datoFom = LocalDate.of(2023, 1, 1),
+                datoTom = LocalDate.of(2023, 3, 31),
+            )
+
+        val perioder =
+            listOf(
+                Beregningsperiode(
```

Comment:
Bra med test her. Kanskje legge på ein periode før etterbetalingsperioden og ein etter også for å få verifisert at dei ikkje blir med?

Reply:
Godt poeng 👍